manifest.toml

```
@@ -0,0 +1,66 @@
+packaging_format = 2
+
+id = "bookstack"
+name = "BookStack"
+description.en = "Platform to create documentation/wiki content"
+description.fr = "Plateforme pour créer du contenu de documentation/wiki"
+
+version = "23.01.1~ynh1"
+
+maintainers = ["eric_G"]
+
+[upstream]
+license = "MIT"
+website = "https://www.bookstackapp.com"
+demo = "https://demo.bookstackapp.com"
+admindoc = "https://www.bookstackapp.com/docs/admin/"
+userdoc = "https://www.bookstackapp.com/docs/user/"
+code = "https://github.com/BookStackApp/BookStack"
+
+[integration]
+yunohost = ">= 11.1.0"
+architectures = "all"
+multi_instance = true
+ldap = true
+sso = false
+disk = "50M"
+ram.build = "50M"
+ram.runtime = "50M"
+
+[install]
+    [install.domain]
+    type = "domain"
+
+    [install.path]
+    type = "path"
+    default = "/bookstack"
+
+    [install.admin]
+    type = "user"
+
+    [install.language]
+    ask.en = "Choose the application language"
+    ask.fr = "Choisissez la langue de l'application"
+    type = "string"
+    choices = ["de", "en", "es_ES", "fa", "fr", "it", "lt", "nl", "pl", "pt_PT", "uk", "zh_CN"]
+    default = "en"
+
+    [install.init_main_permission]
+    help.en = "If enabled, BookStack will be accessible by people who do not have an account. This can be changed later via the webadmin."
+    help.fr = "Si cette case est cochée, BookStack sera accessible aux personnes n’ayant pas de compte. Vous pourrez changer ceci plus tard via la webadmin."
+    type = "group"
+    default = "visitors"
+
+[resources]
+    [resources.system_user]
+
+    [resources.install_dir]
+
+    [resources.permissions]
+    main.url = "/"
+
+    [resources.database]
+    type = "mysql"
+
+    [resources.apt]
+    packages = "php${YNH_PHP_VERSION}-curl php${YNH_PHP_VERSION}-mysql php${YNH_PHP_VERSION}-ldap php${YNH_PHP_VERSION}-mbstring php${YNH_PHP_VERSION}-tidy php${YNH_PHP_VERSION}-xml php${YNH_PHP_VERSION}-zip php${YNH_PHP_VERSION}-gd"
```

So hmmm I thin this one should be:

```suggestion
    packages = "php8.1-curl php8.1-mysql php8.1-ldap php8.1-mbstring php8.1-tidy php8.1-xml php8.1-zip php8.1-gd"
```

(This is a bug from the convert_from_packaging_v1_to_v2 i believe)

Otherwise I think the current behavior is that the var is replaced by empty string and then we see the CI complain that `php-fpm8.1: command not found`

With this change, this should properly also define the `phpversion` setting which then doesnt need to be declared in the install script, hence I would remove the `YNH_PHP_VERSION="8.1"` statement from `_common.sh` as well

However for this to work we'll need https://github.com/YunoHost/yunohost/commit/13d4e16e7de7deab1d4a9e606f5b4ea7b8ebbc0b